app/controllers/decidim/vocdoni/admin/census_controller.rb

```diff
@@ -4,31 +4,24 @@ module Decidim
   module Vocdoni
     module Admin
       class CensusController < Admin::ApplicationController
-        helper_method :elections, :election, :census_path, :status
+        helper_method :elections, :election, :census_path, :status, :census_type
```

Comment:
I think the census_type isn't used in the view, so we should remove it from helper_methods